Makefile

```diff
@@ -35,3 +35,8 @@ python-quality-test:
 
 javascript-quality-test:
 	./node_modules/.bin/eslint ./eox_core/**/*.js*
+
+upgrade: ## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
+	pip install -q pip-tools
```

Comment:
I think that we should create a pip-tools.in and a pip-tools.txt files with this package